buildpacks/nodejs-engine/src/main.rs

```diff
@@ -70,13 +72,16 @@ impl Buildpack for NodeJsEngineBuildpack {
         let inv: Inventory =
             toml::from_str(INVENTORY).map_err(NodeJsEngineBuildpackError::InventoryParseError)?;
 
+        let default_version =
+            Requirement::parse(LTS_VERSION).expect("The default Node.js version should be valid");
+
         let version_range = PackageJson::read(context.app_dir.join("package.json"))
             .map_err(NodeJsEngineBuildpackError::PackageJsonError)
             .map(|package_json| {
                 package_json
                     .engines
                     .and_then(|e| e.node)
-                    .unwrap_or_else(Requirement::any)
+                    .unwrap_or(default_version)
```

Comment:
I presume this change is because before the buildpack installed the latest supported Node.js (which can be a non-LTS release, depending on what stage of the release lifecycle things are at), but moving forwards the buildpack wants to only default to the LTS releases? If so, perhaps worth mentioning this in the PR description? :-)

Reply:
Exactly.  And now it's defaulting to v21.x which we don't want.